CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## v0.129.1, 21 December 2020
+
+- Bump phpstan/phpstan from 0.12.62 to 0.12.63 in /composer/helpers/v2
+- Bump friendsofphp/php-cs-fixer in /composer/helpers/v2
+- Composer: Verify if composer name is compatible with v2
+- Python: Upgrade Python version to 3.9.1 (@ulgens)
+- Bump npm from 6.14.9 to 6.14.10 in /npm_and_yarn/helpers
+- Bump eslint from 7.15.0 to 7.16.0 in /npm_and_yarn/helpers
+- Bump friendsofphp/php-cs-fixer in /composer/helpers/v1
+- Sanitize creds from DependabotError
+- Bump composer/composer from 1.10.16 to 1.10.19 in /composer/helpers/v1
+- Bump phpstan/phpstan from 0.12.62 to 0.12.63 in /composer/helpers/v1
+- Bump pip from 20.3.1 to 20.3.3 in /python/helpers
+- Bump @npmcli/arborist from 2.0.1 to 2.0.2 in /npm_and_yarn/helpers
+- go_modules: limit GitDeps to repo
+- go_modules: fix test warning
+- Bump phpstan/phpstan from 0.12.59 to 0.12.62 in /composer/helpers/v2
```

Comment:
```suggestion
```

Idea: this upgrade is moot because of line :17, just drop it. (Maybe change :17 to `0.12.59 to 0.12.63`